docs/cli-reference/ark_server.md

```diff
@@ -14,9 +14,10 @@ ark server [flags]
 ### Options
 
 ```
-  -h, --help                help for server
-      --log-level           the level at which to log. Valid values are debug, info, warning, error, fatal, panic. (default info)
-      --plugin-dir string   directory containing Ark plugins (default "/plugins")
+  -h, --help                     help for server
+      --log-level                the level at which to log. Valid values are debug, info, warning, error, fatal, panic. (default info)
+      --metrics-address string   the address to expose prometheus metrics (default ":8085")
```

Comment:
Should this be simplified to just port? Or does this allow us to specify a listen address & port (e.g. localhost:8085 or $public_ip:8085)?

Reply:
This does allow you to specify an address. In this case, it will always be `localhost:8085` or just `:8085`.